postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -469,3 +474,19 @@ def _explain_pg_stat_activity(self, rows):
                     1,
                     tags=self._tags + ["error:collect-plan-for-statement-crash"],
                 )
+
+    def _get_query_max_text_size(self):
+        with self._get_db(self._config.dbname).cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
+            self._log.debug("Running query [%s]", MAX_ACTIVITY_QUERY_SIZE_QUERY)
```

Comment:
Not sure this log line is needed. Is there a reason for it?

Reply:
I was just following what seemed to be a pattern in the integration (see [example 1](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/statement_samples.py#L175), [example 2](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/statement_samples.py#L347)). It does seem a bit overkill but I assumed it was judged useful based on prior experiences to include debug logging included SQL queries being executed?

Reply:
It was very helpful for me while debugging so probably worth adding it for these queries as well.